example/ck_tile/03_gemm/gemm_basic.cpp

```diff
@@ -263,7 +262,12 @@ int main(int argc, char* argv[])
                                                                      kPadB,
                                                                      kPadC>;
 
-    using CodegenGemmPipeline = ck_tile::BlockGemmPipelineAGmemBGmemCRegV1<CodegenPipelineProblem>;
+    using CodegemGemmPolicy =
```

Comment:
Codege**m** -> Codege**n**
```suggestion
    using CodegenGemmPolicy =
```